src/textual/css/_style_properties.py

```diff
@@ -909,7 +910,8 @@ def __set__(self, obj: StylesBase, style_flags: Style | str | None):
                             self.name, word, context="inline"
                         ),
                     )
-            style = Style.parse(style_flags)
+            # rich doesn't like "none" mixed with other styles, so short-circuit here.
+            style = Style() if "none" in words else Style.parse(style_flags)
```

Comment:
If I enter `none red` won't that result in an empty Style? Indeed `none` plus any combination would result in an empty style.

I think if `none` is used, then its only valid by itself. It should be an error if there are any other words.

Reply:
`none red` raises an error in rich because the `rich.style.Style.parse` method starts by checking if `none` is alone. Because it isn't, parsing will fail later on because `none` isn't a recognised color.

So, we are mimicking the rich behavior and saying that `none` can only show up by itself?

Reply:
Yes. But with a better error.

A single word in Rich may be a theme color, which is why that happens.

Textual doesn't support Rich's themes, so the styles are a subset of what Rich does. `none` by itself has a special meaning in Textual.

Reply:
Addressed 👍